sdk/lib/cmlib/cmindex.c

```diff
@@ -1285,7 +1285,7 @@ CmpSelectLeaf(IN PHHIVE Hive,
     if (!IndexKey)
     {
         /* Should not happen! */
-        ASSERTMSG("IndexKey = NULL!, it should not happen!\n", FALSE);
+        ASSERTMSG("This should not happen!", IndexKey);
```

Comment:
"This should not happen" is basically the message for an assert without message. Might as well change to `ASSERT(IndexKey != NULL)`... or just leave it as is.

Reply:
Changed to `ASSERT()`.